test/testdrive/mzcompose.yml

```diff
@@ -8,29 +8,95 @@
 # by the Apache License, Version 2.0.
 
 version: '3.7'
+
+mzworkflows:
+  # Run tests with no support for s3
+  local:
+    steps:
+      - step: workflow
+        workflow: start-deps
+      - step: run
+        service: testdrive
+
+  # Run tests with s3 stubbed out, no credentials needed
+  local-aws:
```

Comment:
Can we just call this `testdrive` so that `./mzcompose run testdrive` works as before? (I think just renaming the `testdrive` service to `testdrive-svc` or some such should be sufficient.)

Reply:
sure.

Reply:
I think this will give folks a bit of confusion about what is going on under the hood, but I agree that it does seem more obvious.